modules/integration/src/test/scala/scala/cli/integration/RunJdkTestDefinitions.scala

```diff
@@ -0,0 +1,133 @@
+package scala.cli.integration
+
+import com.eed3si9n.expecty.Expecty.expect
+
+import scala.util.Properties
+
+trait RunJdkTestDefinitions { _: RunTestDefinitions =>
+  def javaIndex(javaVersion: Int): String =
+    // TODO just passing the version number on arm64 should be enough, needs a fix in cs
```

Comment:
Do we have it raised upstream?

Reply:
We don't. 
I'm not yet 100% sure where should we fix it.
Technically speaking, `cs` defaults to adoptium everywhere, so the error it fails with is valid...
```bash
cs java-home --jvm 16 --architecture arm64
# JVM 16 not found in index: No adoptium version matching '1.16+' found
```
I don't think we should fail in Scala CLi under the same circumstances, however.
Perhaps we should just log that the index wasn't found for `adoptium`, and fallback to smth else (i.e. `zulu`)...

Reply:
https://github.com/VirtusLab/scala-cli/issues/3396
raised an issue, let's track it there.